lo/intro/consent-text-lo.html

```diff
@@ -0,0 +1,118 @@
+<div class="consent-title">
+    <center><h4>ນະໂຍບາຍຄວາມເປັນສ່ວນຕົວ/ເງື່ອນໄຂການໃຊ້ງານ NREL OPENPATH</h4></center>
+    </div>
+    <div class="consent-text">
+    <h3>ຈຸດປະສົງຂອງການສຶກສາ</h3>
+    <p>
+    {{template_text.short_textual_description}}
+    </p><p>
+    ຂໍ້​ມູນ​ນີ້​ແມ່ນ​ໄດ້​ຖືກ​ເກັບ​ກໍາ​ໂດຍ​ຜ່ານ OpenPATH​, ເປັນ​ເວ​ທີ​ແຫຼ່ງ​ເປີດ NREL​.
+    ແອັບພລິເຄຊັນໂທລະສັບອັດສະລິຍະ, NREL OpenPATH (“App”), ລວມຂໍ້ມູນຈາກ
+    ເຊັນເຊີໂທລະສັບສະຫຼາດ, ປ້າຍຊື່ຜູ້ໃຊ້ semantic ແລະການສໍາຫຼວດປະຊາກອນສັ້ນ.
+    </p><p>
+    NREL ເປັນຫ້ອງທົດລອງແຫ່ງຊາດຂອງກະຊວງພະລັງງານສະຫະລັດ, ຫ້ອງການຂອງ
+    ພະລັງງານປະສິດທິພາບ ແລະພະລັງງານທົດແທນ, ດໍາເນີນການໂດຍ Alliance for Sustainable
+    ພະລັງງານ, LLC ພາຍໃຕ້ສັນຍາ Prime No. DE-AC36-08GO28308. ນະໂຍບາຍຄວາມເປັນສ່ວນຕົວນີ້
+    ນຳໃຊ້ກັບແອັບທີ່ສະໜອງໃຫ້ໂດຍພັນທະມິດເພື່ອພະລັງງານແບບຍືນຍົງ, LLC. App ນີ້
+    ແມ່ນສະຫນອງໃຫ້ພຽງແຕ່ສໍາລັບຈຸດປະສົງຂອງການເກັບກໍາຂໍ້ມູນພຶດຕິກໍາການເດີນທາງສໍາລັບ
+    {{ui_config.intro.program_or_study}} ແລະເພື່ອການຄົ້ນຄວ້າເພື່ອແຈ້ງນະໂຍບາຍສາທາລະນະ.
+    ບໍ່ມີຂໍ້ມູນໃດໆທີ່ເກັບໄດ້ໂດຍ App ຈະບໍ່ຖືກຂາຍ ຫຼືໃຊ້ສໍາລັບອັນໃດ ຈຸດປະສົງທາງການຄ້າ, ລວມທັງການໂຄສະນາ.
+    </p><p>
+    ຖ້າເຈົ້າບໍ່ເຫັນດີກັບເງື່ອນໄຂຂອງນະໂຍບາຍຄວາມເປັນສ່ວນຕົວນີ້, ກະລຸນາລຶບແອັບອອກ
+    </p>
+    
+    <h3>ເປັນຫຍັງພວກເຮົາເກັບກໍາຂໍ້ມູນນີ້</h3>
+    {{template_text.why_we_collect}}
+    
+    <h3>ການເກັບກຳຂໍ້ມູນ</h3>
+    <p>
+    ລະບົບ ຈະບໍ່ ເກັບກຳຂໍ້ມູນທີ່ສາມາດລະບຸຕົວຕົນໄດ້ແບບດັ້ງເດີມ (ເຊັ່ນ: ບໍ່ມີຊື່, ອີເມວ ຫຼືເບີໂທລະສັບ)
+    </p><p>
+    ລະບົບນີ້ຈະນໍາໃຊ້ ເພື່ອເປັນການເກັບກໍາຂໍ້ມູນໃນການຕິດຕາມ ການເດີນທາງ ເພື່ອໃຫ້ເຂົ້າເຖີງຮູບແບບການເດີນທາງຂອງການນໍາໃຊ້ພາຫະນະ.
+    </p><p>
+    ມັນຍັງຈະຂໍໃຫ້ເຈົ້າບັນທຶກການເດີນທາງທີ່ມີຄວາມຮູ້ສຶກເຫຼົ່ານີ້ເປັນໄລຍະໆດ້ວຍ semantic
+    ປ້າຍຊື່, ເຊັ່ນ: ຮູບແບບການເດີນທາງ, ຈຸດປະສົງ, ແລະຮູບແບບການທົດແທນ.
+    </p><p>
+    It will also request sociodemographic information such as your approximate age,
+    gender, and household type. The sociodemographic factors can be used to
+    understand the influence of lifestyle on travel behavior, as well as generalize
+    the results to a broader population.
+    </p><p>
```

Comment:
Google wasn't good enough?

Reply:
I accidentally got two lines confused earlier, thanks for pointing that out, it's fixed now!